modules/clientmanagements/client/config/clientmanagements.client.routes.js

```diff
@@ -62,8 +62,8 @@
           pageTitle: 'Clientmanagement {{ clientmanagementResolve.name }}'
         }
       })
-      .state('moveinventory', {
-        url: '/moveitems',
+      .state('clientmanagements.moveinventory', {
```

Comment:
this is a change for a different branch, dont put this on this branch. Also, that breaks permissions of the page, I know cause thats what I had initially and I had to change it.